submit/submit

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/env python3
+
+#
+# submit -- command-line submit program for solutions.
+#
+# Part of the DOMjudge Programming Contest Jury System and licensed
+# under the GNU GPL. See README and COPYING for details.
+#
+
+import argparse, sys, logging, os, stat, requests, json, time
+try:
+    import magic
+except ModuleNotFoundError:
+    # Ignore, magic is optional
+    magic = None
+
+# By default no base URL is set and it is expected to use either the
+# SUBMITBASEURL environment variable or the -u/--url argument.
+# If you want to hardcode a default base URL, set it here
+baseurl = ''
+
+# Use a specific API version, set to empty string for default
+api_version = 'v4/'
+
+# Last modified time in minutes after which to warn for submitting
+# an old file.
+warn_mtime = 5
+
+num_warnings = 0
```

Comment:
Add "# End of configurable settings" text above to make it clear that num_warnings is not also something you can tweak